pay-queue/tests/integration/test_eft_reconciliation.py

```diff
@@ -310,11 +304,11 @@ def test_eft_tdi17_fail_transactions(session, app, client, mocker):
     assert "Invalid transaction deposit amount CAD." == call_args[0].error_messages[0]["error"]
 
 
-def test_eft_tdi17_basic_process(session, app, client):
-    """Test EFT Reconciliations worker is able to create basic EFT processing records."""
+def test_eft_tdi17_all_patterns_process(session, app, client):
```

Comment:
refactored to test all configured patterns with short names and generated short names with no defined pattern